docs/interpreter/jdbc.md

```diff
@@ -165,6 +165,10 @@ There are more JDBC interpreter properties you can specify like below.
     <td>The path to the keytab file</td>
   </tr>
   <tr>
+      <td>zeppelin.jdbc.auth.kerberos.proxy.enable</td>
+      <td>true or false. when auth type is kerberos, whether do the kerberos proxy behavior with the login user (if not anonymous) to get the connection, default value is true</td>
```

Comment:
@wary Sorry for my late blocking but when I saw this sentence, I couldn't understand at a glance. Could you make this description:
"when auth type is kerberos, whether do the kerberos proxy behavior with the login user (if not anonymous) to get the connection, default value is true" more understandable? Something like

"When auth type is Kerberos, enable/disable Kerberos proxy with a logged user to get the connection. Default value is true." 

What do you think?